src/client/initClient.jsx

```diff
@@ -63,7 +63,9 @@ export default async function initClient() {
     );
     /* eslint-enable react/jsx-props-no-spreading */
 
-    hydrate(
+    const { __render_mode__: renderMode = 'hydrate' } = global;
```

Comment:
this should probably be a bit more defensive, if `__render_mode__` is somehow set by something else could cause issues
```suggestion
    const renderMode = global.__render_mode__ === 'render'  ?  'render' : 'hydrate'
```

Reply:
I have updated aad162a